src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -756,8 +756,8 @@ string ReserveName(string prefix)
             var additionalDeclarations = new HashSet<string>();
 
             // Declare some locals.
-            string textSpanLocal = "textSpan";
-            writer.WriteLine("string runtext = base.runtext!;");
+            string textSpanLocal = "localSpan";
```

Comment:
runtextSpan is a local as well, so it's a little strange for this one to be called "localSpan".  What if we just call it "slice", since its purpose is to be a subset of runtextSpan?  Open to other ideas as well.

Reply:
yeah, I renamed it because it looked confusing to have `textSpan` and `runtextSpan`, but didn't know which one could be a good name, I also debated on `tempSpan`, but I like `slice` better. I'll make that change.